esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -42,6 +44,8 @@ for more details on the boards see mlrs-wifi-bridge-boards.h
 // Board
 // un-comment what you want
 //#define MODULE_GENERIC
+//#define MODULE_ESP32_DEV
+//#define MODULE_ESP32_DEV_INV
```

Comment:
pl give the flag a more descritive name. With the number of flags more and more growing it becomes more and more difficult to follow through. 
Maybe MODULE_ESP32_DEV_SERIAL_INVERTED

Reply:
Done.